### PR TITLE
Travis CI: Clean up nodejs config from travis file

### PR DESCRIPTION
**Pull request by hariclerry**
### Tickets
Closes #21 

### What does this PR do?
This PR removes nodejs config from .travis.yml file, reverts the file to the previous state, and adds Github actions status badge to the README file

### What are the changes made?
- Remove Node configuration from travis.yml file
- Added previous configurations to the file
- Added Github actions status badge to the Readme file

### Screen shots
None

### CODE REVIEW

File: .github/workflows/node.js.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - run: npm ci
+    - run: npm run lint
+    - run: npm run playwright:silent
+      
+
```
Comment (sovoid) [inline .github/workflows/node.js.yml]: Remove the extra whitespace here

**Review by sovoid — CHANGES_REQUESTED** [review]

File: .github/workflows/node.js.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - run: npm ci
+    - run: npm run lint
+    - run: npm run playwright:silent
```
Comment (sovoid) [inline .github/workflows/node.js.yml]: Add names to the jobs that are verbose. `npm run <script-name>` isnt really explicit about what the script does

**Review by sovoid — CHANGES_REQUESTED** [review]

**Comment by hariclerry** [comment]
@und3fined-v01d , I have made the necessary changes.

File: .github/workflows/node.js.yml [hunk 3]
```diff
@@ -0,0 +1,34 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
```
Comment (sovoid) [inline .github/workflows/node.js.yml:1]: Add this file to a separate pull request

**Review by sovoid — CHANGES_REQUESTED** [review]

**Pull request closed without merging**